app/components/stripe-input/component.js

```diff
@@ -0,0 +1,31 @@
+import Ember from 'ember';
```

Comment:
Lets call this `one-way-input`
* add both `name` and `data-stripe` to `attributeBindings`
* add a `sanitizeValue` method to the component (that is a noop by default) and have `processValue` call it (roughly like handle-input does)
* update `{{handle-input}}` component to extend from this one but provide its custom `sanitizeValue` function

Doing all of that reduces a bit of duplication (and gives us a single place to make fixes to this common portion).